docs/platforms/apple/guides/ios/session-replay/troubleshooting.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Troubleshooting
```

Comment:
`m`: I think we should add a link to this troubleshooting guide from https://docs.sentry.io/platforms/apple/troubleshooting/, so people can find it here.